addons/dragon/functions/script_component.hpp

```diff
@@ -0,0 +1,2 @@
+#include "\z\ace\addons\dragon\script_component.hpp"
+
```

Comment:
No empty line in script_component. Generally not in header files.

Reply:
and now all hpp's have no newline at the end :(